### PR TITLE
Episode 23

### PR DESCRIPTION
**Pull request by sockeqwe**
- Feed is [valid](http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode23/feed.rss)

TODO (I will do that):
- [ ]  add git tag 
- [ ]  upload MP3 file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ Hosts: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), Hannes Dor
 
 ### [Episodes](https://github.com/artem-zinnatullin/TheContext-Podcast/releases)
 
+##### Episode 23: Rise of the Machines
```
Comment (ming13) [inline README.md]: Shouldn’t we mark these episodes though? I. e. _Episode 23: Rise of the Machines (droidcon Berlin 2018)_.

File: show_notes/Episode_23.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* [Jürgen Fey](https://twitter.com/androidian60), Androidian 
+* [Meghan Kane](https://twitter.com/meghafon), Novoda
+* [Hoi Lam](https://twitter.com/hoitab), Google
+* Dr. Roman Belter, APPSfactory
+
+**Panel Host:**
+
+* [Kevin McDonagh](https://twitter.com/kevinmcdonagh), Novoda
+
+Thanks droidcon Berlin and the local Berlin android community for collaborating with us.
```
Comment (ming13) [inline show_notes/Episode_23.md]: _Android community_?

File: feed.rss [hunk 3]
```diff
@@ -1464,6 +1464,43 @@
 				<li><a href="https://de.linkedin.com/in/daniela-gausmann">Daniela Gausmann</a>, ObjectBox</li>
 				</ul>
 
+				<p>Thanks droidcon Berlin and the local Berlin android community for collaborating with us.</p>
+	  ]]></itunes:summary>
+    </item>
+	<item>
+      <guid>https://artemzin.com/static/thecontext/episodes/The.Context.episode.23.mp3</guid>
+      <title>Episode 23: Rise of the Machines</title>
```
Comment (ming13) [inline feed.rss]: The same as above — maybe tag these in the title.

File: README.md (hunk 1 above)
Comment (sockeqwe) [reply README.md]: Sure can do it, but what about the previous already published panel discussions, should we also rename them too?

Comment (ming13) [reply README.md]: I think so. It is a pretty good way to find out when podcast clients update the content (and if they update it at all).

**Comment by sockeqwe** [comment]
Ok, lets do it

Artur Dryomov <notifications@github.com> schrieb am Fr., 21. Sep. 2018,
07:48:

**Comment by sockeqwe** [comment]
Addressed your feedback. thanks for reviewing. Could you do one more final check if and review if its ok now?

**Comment by ming13** [comment]
Looks good! :shipit:

**Pull request merged**